Content.Client/Anomaly/Effects/ClientInnerBodySystem.cs

```diff
@@ -21,21 +23,21 @@ private void OnAfterHandleState(Entity<InnerBodyAnomalyComponent> ent, ref After
         if (ent.Comp.FallbackSprite is null)
             return;
 
-        if (!sprite.LayerMapTryGet(ent.Comp.LayerMap, out var index))
-            index = sprite.LayerMapReserveBlank(ent.Comp.LayerMap);
+        if (!_sprite.LayerMapTryGet((ent.Owner, sprite), ent.Comp.LayerMap, out var index, false))
+            index = _sprite.LayerMapReserve((ent.Owner, sprite), ent.Comp.LayerMap);
```

Comment:
```suggestion
        index = _sprite.LayerMapReserve((ent.Owner, sprite), ent.Comp.LayerMap);
```
Wouldn't this be equivalent? At least once the RT PR fixing the difference between `LayerMapReserveBlank` and `LayerMapReserve` has been merged?

Reply:
I think so? I was wondering about that too. I'll take a look.

Reply:
Yep, looks like it works fine.